tests/python/unittest/test_tir_schedule_blockize.py

```diff
@@ -354,5 +355,168 @@ def after_blocks_blockize(
     verify_trace_roundtrip(sch=s, mod=blocks_func)
 
 
+def test_blockize_blocks_v1():
```

Comment:
Could we depict what this script do, is it an auto-generated script or a certain cv algorithm?

Reply:
This script is only part of implementing cv2.warpAffine